src/Modules/OrchardCore.Commerce/Assets/Styles/shopping-cart-widget.scss

```diff
@@ -0,0 +1,19 @@
+.widget-shopping-cart-widget {
```

Comment:
the Department of Redundancy Department is calling

Reply:
This is intended. It's a css class generated by OC. I'm doing this to avoid leaving a white bar at the top.
![image](https://user-images.githubusercontent.com/80963259/188466586-48bb8c82-0a07-4fc0-a0fd-5abf2839b540.png)

Reply:
Then add a comment above it, e.g. `// Name is generated by OC from the content type.` so people won't wonder about the peculiar naming.